Combined WhoWeAre and Contact Page into About Us page

nsalinas12 opened the pull request.

DheekshuKumar commented:
A few things while you're taking a look at this page anyways:

1.  Add a title that says "Who We Are"  above the tables
2. Move Contact Us to the end
3. Make the font smaller in the Who We Are tables
4. change the span of the Who We Are tables to something that looks prettier on smaller screens :') so idk a span of 10 maybe?

DheekshuKumar commented:
Oh can you also add in a title that actually says "About Us" and modify the "Contact Us" and "Who We Are" headings accordingly so it looks ok?

nsalinas12 commented:
Updated the font sizes and heading levels of the "About Us", "Contact Us"/"Who We Are", and the size of each Who we are table to reflect hierarchy.

Also moved contact to the bottom of the page.

LMK if that address your fourth point about the span in the WWA tables. The font size is 0.9em so its a bit smaller; before it was 1.2em
